projects/module-1/library.md

```diff
@@ -82,11 +82,11 @@ problem.
     different way you could make the tests pass?
 
 
-### Extension (not part of the project, just stuff to do if you get to it): Complete the Librarian tests
-  - Unskip the first test in `test/librarian-test.js`, and get to work passing the tests
-  - Run `npm test test/librarian-test.js`  
-  - This test involves using a class. It will require self-teaching in order to complete.
-  - Ensure that all of the skips are removed from the test file when you push up to GitHub.
+### Extension: Refactoring
+  - Preteach yourself the [Dot & Bracket Notation](https://frontend.turing.edu/lessons/module-1/js-dot-bracket-notation.html){:target="_blank"} lesson.
```

Comment:
Such a meaningful add :)

Reply:
Agreed, this looks great and appreciate you aligning this with the learning goals of JS Fun at the Restaurant!  Looks ready to be merged!